spec/minimax_server/core_spec.clj

```diff
@@ -1,8 +1,8 @@
 (ns minimax-server.core-spec
-  (import [scarvill.httpserver.response ResponseBuilder])
-  (require [speclj.core :refer :all]
+  (:require [speclj.core :refer :all]
             [minimax-server.core :refer :all]))
 
-(describe "temporary tests"
-  (it "can use things from http server jar"
-    (should-be-a ResponseBuilder (new ResponseBuilder))))
+(describe "main"
```

Comment:
That sort of test is definitely a useful stepping stone / temporary test. In fact, I write tests like that all the time. However, they usually don't persist once I write the first test that uses the new class.